rules/RESPONSE-950-DATA-LEAKAGES.conf

```diff
@@ -8,19 +8,23 @@
 # Please see the enclosed LICENSE file for full details.
 # ------------------------------------------------------------------------
 
-# The paranoia level skip rules 950020, 950021 and 950022 have odd
-# numbers not in sync with other paranoia level skip rules in other
-# files. This is done to avoid rule id collisions with CRSv2.
-# This is also true for rule 950130.
+# We reused paranoia level skip rules 950010, 950011, and 950018.
+# These rule identifiers were in use in CRSv2, so we used different
+# identifiers in CRSv3 to avoid conflicts (950020, 950021 and 950022).
+# In CRSv4, for the sake of a consistent structure, we use these
+# identifiers again, so that the skip rules of PL can be defined in the
+# same way for all files.
+# Rule id 950130 has been kept - this ID also was introduced at first
+# rule to avoid the collision.
 
 #
 # -= Paranoia Level 0 (empty) =- (apply unconditionally)
 #
 
 
 
-SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:950020,phase:3,pass,nolog,skipAfter:END-RESPONSE-950-DATA-LEAKAGES"
-SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:950021,phase:4,pass,nolog,skipAfter:END-RESPONSE-950-DATA-LEAKAGES"
+SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:950010,phase:3,pass,nolog,skipAfter:END-RESPONSE-950-DATA-LEAKAGES"
+SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:950011,phase:4,pass,nolog,skipAfter:END-RESPONSE-950-DATA-LEAKAGES"
```

Comment:
Nut sure if this is intended or not but the numbering appears inconsistent with other rule files.

They usually use `*011` and `*012` instead of `*010` and `*011`, for example:
```
SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:951011,phase:3,pass,nolog,skipAfter:END-RESPONSE-951-DATA-LEAKAGES-SQL"
SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:951012,phase:4,pass,nolog,skipAfter:END-RESPONSE-951-DATA-LEAKAGES-SQL"
```

So this should probably be the following instead:
```

SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:950011,phase:3,pass,nolog,skipAfter:END-RESPONSE-950-DATA-LEAKAGES"
SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:950012,phase:4,pass,nolog,skipAfter:END-RESPONSE-950-DATA-LEAKAGES"
```

Reply:
Thanks, that was my mistake. Fixed in [f1957b6](https://github.com/coreruleset/coreruleset/pull/3339/commits/f1957b6ec1f1ec5b2c024db552d3dd403c7d094e).